spec/mail/securityconsiderations.mdown

```diff
@@ -10,7 +10,32 @@ Automatic conversion of charsets that allow hidden channels for ASCII text, such
 
-## Email Submission
+## HTML email display
+
+HTML message bodies provide richer formatting for emails but present a number of security challenges especially when embedded in a webmail context in combination with interface HTML. Clients that render HTML email should make careful consideration of the potential risks, including:
+
+* If JavaScript is allowed to execute it can rewrite email to change its
+  content on subsequent opening, allowing users to be mislead. In webmail systems, if run in the same origin as the interface it can access and exfiltrate all user content accessible via the browser, including all other emails and potentially contacts, calendar events, settings, and credentials. It can also rewrite the interface to undetectably phish passwords. A compromise is likely to be persistent, not just for the duration of page load, due to exfiltration of session credentials or installation of a service worker that can intercept all subsequent network requests (this however would only be possible if blob downloads are also available on the same origin, and the service worker script is attached to the message).
+* HTML documents may load content directly from the internet, rather than just
+  referencing attached resources. For example you may have an `<img>` tag with an external `src` attribute. This may leak to the sender when a message is opened, as well as the IP address of the recipient. Cookies may also be sent and set by the server, allowing tracking between different emails and even website visits and advertising profiles.
+* In webmail systems, CSS can break the layout or create phishing
+  vulnerabilities. For example, the use of `position:fixed` can allow an email to draw content outside of its normal bounds, potentially clickjacking a real interface element.
+* If not inside a separate frame in a webmail context, any styles defined in
+  CSS rules will apply to interface elements as well if the selector matches, allowing the interace to be modified. Similarly, any interface styles that match elements in the email will alter their appearance, potentially breaking the layout of the email.
+* The link text in HTML has no neccessary correlation with the actual target of
+  the link, which can be used to make phishing attacks more convincing.
+* Links opened from an email may leak private info in the `Referer` header sent
+  by default in most systems. This can be mitigated with the use of a [referrer policy](https://www.w3.org/TR/referrer-policy/).
+
+Sanitizing HTML to mitigate these issues is tricky. Using a well-tested library with a carefully vetted whitelist-only approach is the safest way to avoid vulnerabilities. New features with unexpected security characteristics may be added to HTML rendering engines in the future; a blacklist approach is likely to result in security issues.
```

Comment:
:+1: I suppose, in a spec, you want to be neutral. If not, I'd recommend DOMPurify :)